specification/paths/Returns-v1-returns-allow-reissue.json

```
@@ -0,0 +1,68 @@
+{
+  "post": {
+    "tags": [
+      "Returns"
+    ],
+    "security": [
+      {
+        "OAuth2": [
+          "returns.manage"
+        ]
+      }
+    ],
+    "summary": "Allow an order reference to be reissued as a return.",
+    "description": "This endpoint allows a return to be reissued for an existing order reference.",
+    "requestBody": {
+      "description": "The shop ID and the order reference.",
+      "required": true,
+      "content": {
+        "application/vnd.api+json": {
+          "schema": {
+            "type": "object",
+            "required": [
+              "data"
+            ],
+            "additionalProperties": false,
+            "properties": {
+              "data": {
+                "type": "object",
+                "required": [
+                  "shop_id",
+                  "order_reference"
+                ],
+                "additionalProperties": false,
+                "properties": {
+                  "shop_id": {
+                    "type": "string",
+                    "format": "uuid",
+                    "pattern": "^[a-f\\d]{8}(-[a-f\\d]{4}){3}-[a-f\\d]{12}$",
+                    "example": "4067e35b-09cc-42b9-8553-6eaf1788df35"
+                  },
```

We have a `Uuid` schema to reference
```suggestion
                  "shop_id": {
                    "$ref": "#/components/schemas/Uuid"
                  },
```
but it does not include an example, so to add one, I believe you would have to make an `allOf` construction, which is a bit verbose...
```suggestion
                  "shop_id": {
                    "allOf": [
                      {
                        "$ref": "#/components/schemas/Uuid"
                      },
                      {
                        "example": "4067e35b-09cc-42b9-8553-6eaf1788df35"
                      }
                    ]
                  },
```
I'll leave it up to you if you want to make any changes